src/httpserver.cpp

```diff
@@ -93,9 +93,10 @@ class WorkQueue
 
 public:
     WorkQueue(size_t _maxDepth) : running(true),
-                                 maxDepth(_maxDepth),
-                                 numThreads(0)
+                                 maxDepth(_maxDepth)
     {
+        std::lock_guard<std::mutex> lock(cs);
```

Comment:
The way forward would be to fix the tools, not contort our code around whatever happen to be the bugs in the analysis tools of the day.